graphql/task_resolver.go

```diff
@@ -120,12 +120,12 @@ func (r *taskResolver) BaseTask(ctx context.Context, obj *restModel.APITask) (*r
 		if evergreen.IsPatchRequester(t.Requester) {
 			baseTask, err = t.FindTaskOnBaseCommit()
 			if err != nil {
-				return nil, InternalServerError.Send(ctx, fmt.Sprintf("Error finding task %s on base commit: %s", *obj.Id, err.Error()))
+				return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding task %s on base commit: %s", *obj.Id, err.Error()))
```

Comment:
`utility.FromStringPtr`
```suggestion
				return nil, InternalServerError.Send(ctx, fmt.Sprintf("finding task '%s' on base commit: %s", *obj.Id, err.Error()))
```